### DISCUSSION TITLE
Latest version missing Node-RED Manage Palette

### DISCUSSION THREAD

**Discussion by aperture1988** [zachowj/node-red-contrib-home-assistant-websocket#391] [Q&A]
Version 0.31.0 doesn't seem to be available to install via the Node-RED Manage Palette, 0.30.1 is listed as the available version.

thanks

**Comment by zachowj** [comment]
https://flows.nodered.org/node/node-red-contrib-home-assistant-websocket

It's there, your system probably just hasn't checked for an update yet.
